packages/core/src/sink/SafeSink.ts

```diff
@@ -1,34 +1,38 @@
 /** @license MIT License (c) copyright 2010-2016 original author or authors */
 /** @author Brian Cavalier */
 /** @author John Hann */
+import { Sink, Time } from '@most/types'
 
-export default class SafeSink {
-  constructor (sink) {
+export default class SafeSink<A> implements Sink<A> {
+  private readonly sink: Sink<A>
+  active: boolean;
```

Comment:
Should consumers know about the `active` state? Even being able to manipulate it? If not, it should likely be protected (or private, which I am not a huge fan of.) It is always possible to open a protected or private field using accessors and mutators. @briancavalier, I need you to chime in on this.

Reply:
Looks like it's accidental, changed to `private`